features/step_definitions/mobile_steps.rb

```diff
@@ -33,3 +33,7 @@
 When /^I open the drawer$/ do
   find('#menu_badge').click
 end
+
+Then /^the aspect dropdown should be labeled "([^"]*)"/ do |label|
+  expect(page).to have_css("#user_aspects option.list_cover", :text => label)
```

Comment:
No, there I could have used `.user_aspects`. This is where I could not use the class: https://github.com/diaspora/diaspora/pull/5594/files#diff-07ff6d8f76a7cabb01ec40c8990d498bR19